src/main/groovy/de/tracetronic/jenkins/plugins/ecutestexecution/steps/AbstractProvideExecutionFilesStep.groovy

```diff
@@ -57,8 +57,14 @@ abstract class AbstractProvideExecutionFilesStep extends Step implements Seriali
     }
 
     @DataBoundSetter
-    void setReportIds(List<String> reportIds) {
-        this.reportIds = reportIds ? StepUtil.removeEmptyReportIds(reportIds): []
+    void setReportIds(def reportIds) {
+        if (reportIds instanceof String) {
+            this.reportIds = reportIds ? reportIds.split(",")*.trim().findAll { it } : []
```

Comment:
`this.reportIds = StepUtil.removeEmptyReportIds(reportIds.split(","))` should do the same